test/common/fixtures.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable required-modules */
+'use strict';
+
+const path = require('path');
+const fs = require('fs');
+
+const fixturesDir = path.join(__dirname, '..', 'fixtures');
+
+function fixturesPath(...args) {
+  return path.join(fixturesDir, ...args);
+}
+
+function readFixtureSync(args, enc) {
+  if (Array.isArray(args))
+    return fs.readFileSync(fixturesPath(...args), enc);
+  return fs.readFileSync(fixturesPath(args), enc);
+}
+
+function readFixtureKey(name, enc) {
+  return fs.readFileSync(fixturesPath('keys', name), enc);
```

Comment:
I'm not saying they're identical, I'm saying that `fixtures.readKey()` is a specialised version of `fixtures.readSync()` (the one where the first arg is `keys` and you can omit the `encoding`).

You're right, I should have said:

```js
fixtures.readKey('agent1-key.pem') == fixtures.readSync(['keys', 'agent1-key.pem'], 'utf8')
```

For me the latter is clearer (which is why I don't understand the need for `readFixtureKey()`)